### PR TITLE
NEWS: version 3.21

### PR DESCRIPTION
**Pull request by MrAnno**
TODO: 

- [x] #2651
- [x] #2700
- [x] #2701

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by kira-syslogng** [comment]
Build SUCCESS

File: NEWS.md [hunk 1]
```diff
@@ -1,90 +1,83 @@
...
 ## Features
 
- * Add persist-tool (#2511)
- * Collectd destination (#1433)
- * Config reload status feedback (#2367)
- * Netskope parser (#2541)
- * Websense parser (#2471)
- * Json list support (#2536)
- * Xml-parser: add list support (#2544)
+ * Rdkafka destination (#2496)
```
Comment (bazsi) [inline NEWS.md]: I would be a more verbose here, like:

"Adding an alternative, native, librdkafka based kafka-c() destination in parallel of the existing Java implementation, that provides the same configuration interface. Eventually, we expect this to replace the Java one."

File: NEWS.md [hunk 2]
```diff
@@ -1,90 +1,83 @@
...
+ * Added elasticsearch-http() destination (#2509)
```
Comment (bazsi) [inline NEWS.md]: Added a native, http() destination based driver for elasticsearch, as an alternative of the Java one. Eventually, we expect this to replace the Java implementation.

File: NEWS.md [hunk 3]
```diff
@@ -1,90 +1,83 @@
...
+ * Trim large messages (#2644)
```
Comment (bazsi) [inline NEWS.md]: The ability to work with messages larger than log-msg-size(), previously the messages longer than the limit were split into two segments, now syslog-ng supports in truncating the oversize part.

**Comment by bazsi** [comment]
I have now updated the branch with my improvements and also added the two outstanding items to the file.

@fekete-robert a review would be useful but not required.

**Comment by fekete-robert** [comment]
I think it's fine, I've added the features I didn't know about to my
documentation todo list

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by MrAnno** [comment]
@bazsi Thanks!

The verbose version looks really good! :+1:

**Comment by gaborznagy** [comment]
I've added #2701 to the News file.
I've squashed the changes and pushed it to the branch.

**Comment by gaborznagy** [comment]
I've skipped adding a PR, #2531 which does not have a user-facing change, but a performance improvement.

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Pull request merged**